README.md

```diff
@@ -7,14 +7,42 @@ The configurations files can be one of several formats, such as Terraform, JSON,
 There is a built-in set of rules provided for Terraform. Custom files are used
 for other formats.
 
-# Installation 
+# Installation
+
+There are three main ways that you can install `config-lint`
+
+* homebrew
+* docker
+* manually
+
+## Homebrew
+
 You can use [Homebrew](https://brew.sh/) to install the latest version:
 
-```
+``` bash
 brew tap stelligent/tap
 brew install config-lint
 ```
 
+## Docker
+
+You can pull the latest image from [DockerHub](https://hub.docker.com/r/stelligent/config-lint):
+
+``` bash
+docker pull stelligent/config-lint
+```
+
+Note that if you choose to install and run via `docker` then you will need mount a directory to the running container so that it has access to your configuration files.
+
+``` bash
+cd /path/to/your/configs/
+docker run -v $(pwd):/foobar stelligent/config-lint -terraform /foobar/foo.tf
+--- or ---
+docker run --mount src="$(pwd)",target=/foobar,type=bind stelligent/config-lint -terraform /foobar/foo.tf
+```
+
+## Manually
+
```

Comment:
Can I ask either @akuma12 or @tmcelhattan to double-check these steps in a naive way?

Just looking for a way to "QA" this documentation.

Reply:
``` bash
➜  config git:(feature/deploy_to_dockerhub_issue_34) ✗ pwd
/Users/peter.helewski/Stelligent/config-lint/example-files/config
```
``` bash
➜  config git:(feature/deploy_to_dockerhub_issue_34) ✗ docker run --mount src="$(pwd)",target=/custom,type=bind stelligent/config-lint -terraform /custom/elb.tf
[
  {
    "AssertionMessage": "access_logs should be present",
    "Category": "resource",
    "CreatedAt": "2020-01-13T14:54:52Z",
    "Filename": "/custom/elb.tf",
    "LineNumber": 45,
    "ResourceID": "elb_no_access_logs",
    "ResourceType": "aws_elb",
    "RuleID": "ELB_ACCESS_LOGGING",
    "RuleMessage": "ELB should enable access logging",
    "Status": "WARNING"
  }
]
```
``` bash
➜  config git:(feature/deploy_to_dockerhub_issue_34) ✗ docker run -v $(pwd):/custom stelligent/config-lint -terraform /custom/elb.tf
[
  {
    "AssertionMessage": "access_logs should be present",
    "Category": "resource",
    "CreatedAt": "2020-01-13T14:55:30Z",
    "Filename": "/custom/elb.tf",
    "LineNumber": 45,
    "ResourceID": "elb_no_access_logs",
    "ResourceType": "aws_elb",
    "RuleID": "ELB_ACCESS_LOGGING",
    "RuleMessage": "ELB should enable access logging",
    "Status": "WARNING"
  }
]
```

Reply:
Can confirm I get the same results. Looks good. I didn't have the image built locally and it built and ran with no trouble.